wavelength-plugins/exp.js

```diff
@@ -64,9 +64,10 @@ function addExp(user, room, amount) {
 				let reward = '';
 				switch (level) {
 				case 5:
-					Economy.logTransaction(user.userid + ' received a custom symbol for reaching level ' + level + '.');
-					user.canCustomSymbol = true;
-					reward = 'a Custom Symbol. To claim your custom symbol, use the command /customsymbol [symbol]';
+					Economy.logTransaction(user.userid + ' received a profile background and profile music for reaching level ' + level + '.');
+					WL.messageSeniorStaff(user.userid + ' has earned a profile background and profile music for reaching level ' + level + '.');
```

Comment:
Instead of PMing upper staff, lets just log to staff (which we do below).

Reply:
Ok